docs/pvcscale.md

```diff
@@ -0,0 +1,179 @@
+
+PVCScale workload is designed to stress storage backend from the scale point of
+view. It will create desired number of pods and attach an PVC to each pod.
+During this test we measure time necessary for pods to be in `Ready/Ready` state
+and for all PVC to be in `Bound` state.
+
+For PVCScale test to work it is necessary that cluster is operational and there
+is working storage class which will be used to carve PVCs from. If storageclass
+is not present then this test will fail and will not work.
+
+Running from CLI:
+
+```sh
+$ cp workloads/inventory.example inventory
+$ # Add orchestration host to inventory
+$ # Edit vars in workloads/vars/pvcscale.yml or define Environment vars (See below)
+$ time ansible-playbook -vv -i inventory workloads/pvcscale.yml
+```
+
+## Environment variables
+
+### PUBLIC_KEY
+Default: `~/.ssh/id_rsa.pub`
+Public ssh key file for Ansible.
+
+### PRIVATE_KEY
+Default: `~/.ssh/id_rsa`
+Private ssh key file for Ansible.
+
+### ORCHESTRATION_USER
+Default: `root`
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+Default: `true`
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+Default: `true`
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+Default: `true`
+Enables/disables running the workload pod as privileged.
+
+### KUBECONFIG_FILE
+Default: `~/.kube/config`
+Location of kubeconfig on orchestration host. PVCScale tests supports only
+default at this time.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+Default: `~/.ssh/id_rsa`
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+Default: `~/.ssh/id_rsa.pub`
+Location of the ssh public key to authenticate to the pbench results server.
+
+### ENABLE_PBENCH_AGENTS
+Default: `false`
+Enables/disables the collection of pbench data on the pbench agent pods. These pods are deployed by the tooling playbook.
+
+### PBENCH_SERVER
+Default: There is no public default.
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+Default: There is no public default.
+Future use for pbench and prometheus scraper to place results into git repo that holds results data.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+Default: `360`
+Number of retries for Ansible to poll if the workload job has completed. Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+
+### PVCSCALE_TEST_PREFIX
+Default: `pvcscale`
+Test to prefix the pbench results.
+
+### PVCSCALE_CLEANUP
+Default: `true`
+Enables/disables cleanup of this workload on completion. Cleanup in
+PVCScale terminology means delete `pvcscale0` projects - this operation can be
```

Comment:
"means deletion of"?
projects &mdash; or &ndash; here depending on your preference?